tests/core/pyspec/eth2spec/test/helpers/rewards.py

```diff
@@ -0,0 +1,169 @@
+from random import Random
+
+from eth2spec.test.helpers.attestations import prepare_state_with_full_attestations
+from eth2spec.utils.ssz.ssz_typing import Container, uint64, List
+
+
+# HACK to get the generators outputting correctly
```

Comment:
@protolambda I had a bit of trouble getting generators to output the deltas arrays as I wanted.

`yield rewards` would yield each of the elements of the list as it's own yaml file -- e.g. `rewards_0.yaml`, `rewards_1.yaml`, etc.

Whereas `yield [rewards]` led to a `ruamel.yaml.representer.RepresenterError: cannot represent an object` error. I think it had to do with the SSZ integer type not being able to be parsed properly. I tried rendering it as dictionaries too and had similar problems.

Putting it in a temporary wrapper `Delta` SSZ class rendered properly, but also renders an unnecessary `.ssz` version of the lists.  Curious if you are familiar with the proper method here.

Reply:
> Whereas yield [rewards] led to a ruamel.yaml.representer.RepresenterError: cannot represent an object error.

It can't guess what type of SSZ list that was supposed to be. `[]` in the spec is coerced all the time to the type of the field it's being set to. Here there's no destination, just encoding result.

Maybe we should just have a `Deltas = List[uint64, 2**30]`, or combine rewards and penalties like;
```
class Deltas(Container):
    rewards: List[uint64, 2**30]
    penalties: List[uint64, 2**30]
```

Also, I think the .ssz is good to have. If we have lots of reward deltas to parse (that's the intention), it's just faster, and easier to integrate for some clients.

Reply:
went with `Deltas` class. Thanks!